### PR TITLE
CoronaTest refactoring (EXPOSUREAPP-12330)

### PR DESCRIPTION
**Pull request by chiljamgossow**
Split interface to facilitate differentiation between personal and family tests 

changes come from renaming, logic stays the same

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/CoronaTestRepository.kt [hunk 1]
```diff
@@ -59,19 +59,19 @@ class CoronaTestRepository @Inject constructor(
     /**
      * Returns a flow with an unfiltered set of [CoronaTest]
```
Comment (DerVogel2020) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/CoronaTestRepository.kt:60]: The docs should be changed as well.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/CoronaTest.kt [hunk 2]
```diff
@@ -49,10 +29,16 @@ interface CoronaTest : Recyclable {
     // Has the corresponding entry been created in the test certificate storage
     val isDccDataSetCreated: Boolean
 
-    //  The ID of the lab that uploaded the test result
-    val labId: String?
-
     val qrCodeHash: String?
+}
+
+interface KeySubmission {
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/CoronaTest.kt:35]: Maybe we should be more specific here as it is for test `CoronaTestKeySubmission` or `CoronaTestSubmission` ?

**Review by mtwalli — APPROVED** [review]
Nice 👌, one comment , which can be addressed in followup PR

**Review by SamuraiKek — APPROVED** [review]
LGTM.

**Pull request merged**